charts/seed-controlplane/charts/dependency-watchdog/templates/configmap.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dependency-watchdog-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: dependency-watchdog
+data:
+  dep-config.yaml: |-
+      service: {{ .Values.dependencywatchdog.service.name }}
+      namespace: {{ .Release.Namespace }}
+      labels:
+{{ toYaml .Values.dependencywatchdog.service.labels | indent 8 }}
+      dependantpods:
```

Comment:
Sorry. I missed in the `dependency-watchdog repo. Shouldn't this be `dependantPods`?

Reply:
Json deserializer uses `dependantpods` field [here](https://github.com/gardener/dependency-watchdog/blob/master/pkg/restarter/types.go#L52).